src/helpers/render-document-file.js

```diff
@@ -141,7 +143,19 @@ export const buildList = async (vNode, docxDocumentInstance, xmlFragment) => {
           } else {
             const paragraphVNode = new VNode(
               'p',
-              null,
+              isVNode(childVNode) && childVNode.properties && parentVNodeProperties
+                ? {
+                    attributes: {
+                      ...(parentVNodeProperties.attributes || {}),
+                      ...(childVNode.properties.attributes || {}),
+                    },
+                    style: {
+                      ...(parentVNodeProperties.style || {}),
+                      ...(childVNode.properties.style || {}),
+                    },
+                  }
+                : null, // copy properties for styling purposes
```

Comment:
what is the whole piece for? copying list styles to the paragraph node contained inside li?

Reply:
This is how we add the parent LI styled in. It copies it top down where the children take precedence. The empty objects are safeguards.

Reply:
cool, thanks for the clarification